modules/gateway/nodes.go

```diff
@@ -18,6 +18,18 @@ var (
 	errPeerGenesisID = errors.New("peer has different genesis ID")
 )
 
+// A node represents a potential peer on the Sia network. The Prioritize field
+// indicates whether the node should be prioritized when forming new
+// connections.
+type node struct {
+	NetAddress modules.NetAddress `json:"netaddress"`
+	Prioritize bool               `json:"prioritize"`
```

Comment:
"Outbound" feels vague to me. What is an "outbound node?" What we want to express is: "A node that we, at some point, successfully initiated a connection to and retained as a peer." But I couldn't come up with a terse way of saying that. `WasOutboundPeer` is wordy but at least more accurate.

If we do go with `Oubound`, I think we should also convert the `Inbound` field of the `peer` object` to `Outbound`.